CMakeLists.txt

```diff
@@ -69,7 +78,7 @@ if(RE2C)
 	re2c(${PROJECT_SOURCE_DIR}/src/lexer.in.cc ${PROJECT_BINARY_DIR}/lexer.cc)
 	add_library(libninja-re2c OBJECT ${PROJECT_BINARY_DIR}/depfile_parser.cc ${PROJECT_BINARY_DIR}/lexer.cc)
 else()
-	message(WARNING "re2c was not found; changes to src/*.in.cc will not affect your build.")
+	message(WARNING "re2c was not found/used; changes to src/*.in.cc will not affect your build.")
```

Comment:
`re2c <our version number> or later was not found; changes to src/*.in.cc will not affect your build.`